.eslintrc

```diff
@@ -0,0 +1,80 @@
+{
+  "env": {
+    "browser": true,
+    "jquery": true
+  },
+  "globals": {
+    "fluid_2_0_0": true,
+  },
+  "rules": {
+    "block-scoped-var": "error",
+    "comma-style": [
+      "error",
+      "last"
+    ],
+    "curly": [
+      "error",
+      "all"
+    ],
+    "dot-notation": [
+      "error",
+      {
+        "allowKeywords": false
+      }
+    ],
+    "eqeqeq": [
+      "error",
+      "allow-null"
+    ],
+    "indent": ["error", 4],
+    "new-cap": ["error", { "properties": false }],
+    "no-caller": "error",
+    "no-cond-assign": [
+      "error",
+      "except-parens"
+    ],
+    "no-debugger": "error",
+    "no-empty": ["error", {"allowEmptyCatch": true}],
+    "no-eval": "error",
+    "no-extend-native": "error",
+    "no-extra-parens": "off",
+    "no-irregular-whitespace": "error",
+    "no-iterator": "error",
+    "no-loop-func": "error",
+    "no-multi-str": "error",
+    "no-new": "error",
+    "no-plusplus": 0,
+    "no-proto": "error",
+    "no-script-url": "error",
+    "no-sequences": "error",
+    "no-trailing-spaces": "error",
+    "no-undef": "error",
+    "no-unused-vars": "error",
+    "no-with": "error",
+    "quotes": [
+      "error",
+      "double"
+    ],
+    "semi": [
+      0,
+      "always"
+    ],
+    "space-before-blocks": ["error", "always"],
+    "space-before-function-paren": ["error", {"anonymous": "always", "named": "never"}],
+    "space-infix-ops": "error",
+    "space-unary-ops": [
+      "error", {
+        "words": true,
+        "nonwords": false,
+        "overrides": {
+            "typeof": false
+        }
+    }],
+    "strict": ["error", "function"],
+    "valid-typeof": "error",
+    "wrap-iife": [
+      "error",
+      "inside"
+    ]
+  }
+}
```

Comment:
Yes, I am. It's annoying to see a red warning on the git diff UI when EOFs are missing. Thanks for adding that. 👍